src/btree/bt_vrfy.c

```diff
@@ -311,23 +311,13 @@ __verify_addr_ts(WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK *unpack,
 {
     char ts_string[2][WT_TS_INT_STRING_SIZE];
 
-    if (unpack->newest_stop_ts == WT_TS_NONE)
```

Comment:
These have to go. I thought I may be able to keep them by moving the logic to a higher level, however, if we reconcile the page then these "none" transaction ids and timestamps will actually get written to the disk. So I don't think we can assert on anything useful here.

Reply:
I think the case to consider is a table where timestamps aren't being used.  We might still have a range of transaction IDs that need to be retained, so when we evict there could be a tombstone that isn't obsolete (it has an active `stop_txn` but `stop_ts == WT_TS_NONE`).  That value shouldn't be visible after a restart but could be reconciled then read back into the same process.

Reply:
> As a possible change, did you consider using some other value than `none`, creating a valid window that would be ignored in all cases, like (0,1)?

ba76501

Change to use timestamp 1 instead. We can't do the same for transaction ids but it's nice that we can at least keep some of these checks.

> I'm confused how these values get written to disk -- shouldn't reconciliation simply discard these pages, doesn't it mean there's nothing useful on them?

If a page with a previous startup generation gets read in, some keys are modified and it gets reconciled again, any cells for values that did NOT get modified could have `stop_txn == WT_TXN_NONE` and these time pairs will actually get written to the disk. So the next time it gets read in, your write gen says it is from the current run however you'll see these corrected time pairs.